engine/src/flutter/shell/platform/darwin/macos/BUILD.gn

```diff
@@ -18,6 +18,31 @@ shell_gpu_configuration("macos_gpu_configuration") {
   enable_metal = true
 }
 
+config("config_macos") {
+  include_dirs = [ target_gen_dir ]
+  lib_dirs = mac_swift_lib_paths
+
+  # Allow use of @testable imports in debug builds and when tests are enabled.
+  if (enable_ios_unittests) {
```

Comment:
Should this be `enable_unittests`?

Reply:
It should indeed! We should probably have something like `enable_host_unittests` and `enable_target_unittests`, but (currently) host builds happen to build the desktop unittests and `enable_unittests` conflates the two for (some, i.e. macOS and Fuchsia, but not Linux and Windows) desktop builds anyway.

That's a separate cleanup we should probably do. Or we declare a `enable_${target_os}_unittests` for each `target_os` we support to avoid `enable_target_unittests && is_ios` (etc.). I dunno. Needs some thought I think.